### PR TITLE
Improvements/Updates

### PR DESCRIPTION
**Pull request by plowsof**
- add default material theme config options
- add privacy + tables plugin
- add i18n plugin + locales
- replace 404.html with template
- pip freeze > requirements.txt
- add dockerfile for coolify previews

TODO: 

- [x] add the previous favicon (perhaps as a placeholder) https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#favicon
- [x] fix site/repo urls to point to... test branch (for now)
- [ ] copyright and what not

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -58,41 +58,68 @@ nav:
...
+  favicon: assets/favicon.png
   icon:
     logo: logo
   logo: 'images/monero.svg'
 
 extra_css: ['overrides.css']
 
-repo_name: 'monerodocs/md'
-repo_url: 'https://github.com/monerodocs/md'
+repo_name: 'plowsof/monero-docs'
```
Comment (nahuhh) [inline mkdocs.yml:120]:
```suggestion
repo_name: 'monero-project/monero-docs'
```

File: mkdocs.yml [hunk 2]
```diff
@@ -58,41 +58,68 @@ nav:
...
+repo_url: 'https://github.com/plowsof/monero-docs'
```
Comment (nahuhh) [inline mkdocs.yml:121]:
```suggestion
repo_url: 'https://github.com/monero-project/monero-docs'
```

File: mkdocs.yml [hunk 3]
```diff
@@ -58,41 +58,68 @@ nav:
...
+edit_uri: blob/test_test/docs/
```
Comment (nahuhh) [inline mkdocs.yml:122]:
```suggestion
edit_uri: blob/test/docs/
```

need 2 change to master if/when we go to master branch

File: mkdocs.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 site_name: Monero Documentation
 site_description: 'Unofficial Monero Documentation'
 site_author: "Piotr 'Qertoip' Włodarek"
-site_url: 'https://monerodocs.org'
+site_url: 'https://md2.monerodevs.org'
```
Comment (nahuhh) [inline mkdocs.yml:5]:
```suggestion
site_url: 'https://docs.getmonero.org'
```

**Review by nahuhh — APPROVED** [review]

**Review by 0xFFFC0000 — APPROVED** [review]
LGTM

**Pull request merged**